src/contents/articles/cold-drinks-were-the-remedy.md

```diff
@@ -0,0 +1,22 @@
+---
+title: Cold Drinks Were the Remedy
+author: msanatan
+date: 2015-06-29
+template: article.jade
+---
+
+What do Code and Drinks have in common? Not that much thankfully (imagine the bugs). Since we had an event for [coding](/articles/public-coding-roundup.html) recently, this time we got about the drinking part!
+<span class="more"></span>
+
+We got cozy at [Bootleggers](https://www.facebook.com/Bootleggerstt) from around 6pm. The rain killed our dream to sit outside but it didn't kill night one bit. 
+
+Our conversations flowed from the get go. At times we were looking at software developer jobs available in T&T, and how to ensure all our good talent doesn't get exported. Some Node.js developers had a chat about the future of JavaScript (we're pretty excited for ES6!) and how [NPM](https://www.npmjs.com/) really supports an ecosystem that keeps developers happy and productive. We weren't just forward thinking for Node.js, what about Software Development as a whole? The region seems to be blazing with mobile app development, maybe data might be the next big wave to take us by storm? No one seemed sure but we all got the feeling that even bigger things are happening - it's an amazing time to be in our profession. 
+
+How much code do you have to write to buy a brand new car? For one guy it took 18 lines (what am I doing with my life?). Hey, that guy doesn't like >insert programming style preference<! No one cares, we had a lot bigger things on the table. Speaking of things on the table, the drinks were not a myth! I elected for the following beauty:
```

Comment:
What is this story about 18 lines of code? If you want to mention the story expand on it or give a link. Additionally, you haven't stated why it is it relevant to what happened at the event. Honestly, you didn't have anything bigger if you're referring to drinks and food.